### ISSUE TITLE
ESP32-C3 Support I2S TDM？

### ISSUE THREAD

**Issue by 1998CN** [espressif/esp-idf#7614]
in the datasheet, ESP32-C3 can use I2S TDM. have any example refer to？

**Comment by L-KAYA** [comment]
Yes, it supports. As for the example, there has no TDM example yet, but later there will have the TDM mode unit-test in `i2s_test.c`, you can refer to that case then.

Thanks for your advice, we'll wrote an example for TDM mode soon.
